routes/alumni.js

```diff
@@ -4,10 +4,12 @@ var _ = require('lodash')
 const alumni = require('../models/alumni')
 require('../models/class')
 require('../models/language')
+const auth = require('../middleware/middleware')
```

Comment:
any reason for importing this middleware?

Reply:
There is no reason for the middleware to be there. Was just there when I was testing the middleware. Has been removed.